### PR TITLE
Add input files and tests for CO2 constrained scenarios.

### PR DESCRIPTION
**Pull request by rplzzz**
I added these because once we turn our attention back to the hector calibration work, we will want to run concentration-constrained scenarios.  Having configuration files for those runs provided makes that work easier to get moving, and testing the capability is a good idea anyhow.

In the course of putting this together, I found that we don't actually have concentration constraint input files for the RCP26 or RCP60 scenarios, so we will need to track down and add that data eventually.

### CODE REVIEW

**Comment by kdorheim** [comment]
@cahartin these commits should take care of your review comments

File: inst/input/hector_rcp26_constrained.ini  [hunk 1]
```diff
@@ -0,0 +1,387 @@
+; Config file for hector model: RCP26
```
[the remark this answers is not in the document]
Comment (kdorheim) [reply inst/input/hector_rcp26_constrained.ini ]: 👍

**Comment by kdorheim** [comment]
@bpbond Hmm I am a little confused about the Travis error that is being thrown here. It passes tests and check on my local machine. It looks like it is registering package check warnings as errors

**Comment by bpbond** [comment]
@kdorheim I would guess that it's a too-long filename–see the [CRAN docs on this topic](https://cran.r-project.org/doc/manuals/R-exts.html#Package-structure). Try shortening "hector_rcp26_constrained.ini" to "rcp26_constr.ini" or something like that.

**Pull request merged**